info.femm.unofficial-manifest.yml

```diff
@@ -0,0 +1,235 @@
+app-id: info.femm.unofficial
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: femm.sh
+build-options:
+  env:
+    - "WINEPREFIX": "/var/data/.local/share/femm42-flatpak"
+    - "WINEDLLOVERRIDES": "mscoree,mshtml="
+    - "V": "1"
+
+#rename-desktop-file: wine.desktop
+
+# Support 32-bit at runtime
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    versions: '21.08'
+    add-ld-path: lib
+    merge-dirs: lxvst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '21.08'
+
+# Support 32-bit at buildtime
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --env=WINEDEBUG=-all
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --allow=multiarch
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.Mutter.DisplayConfig
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
```

Comment:
this is really a broad permission.

Reply:
filesystem=host is really bothering me as well. The alternative is to use portal, which, I know nothing. 
In general, open and save simulated files is the goal.

About Notifications, DisplayConfig and Flatpak. Should I omit it?

Reply:
I would remove them all and just add what is needed.